package.json

```diff
@@ -84,15 +84,16 @@
     "@rollup/plugin-node-resolve": "^15.2.3",
     "@rollup/plugin-replace": "^5.0.7",
     "@rollup/plugin-terser": "0.4.4",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^11.2.7",
-    "@testing-library/react-hooks": "^3.4.2",
-    "@types/react": "^18.2.52",
-    "@vitest/coverage-v8": "^1.2.2",
+    "@testing-library/jest-dom": "^6.4.8",
+    "@testing-library/react": "^12.1.5",
+    "@testing-library/react-hooks": "^8.0.1",
+    "@types/jest": "^29.5.12",
+    "@types/react": "^16.14.60",
```

Comment:
Downgrade @types/react to v16 was necessary because of @testing-library/react-hooks v8. Regardless, it makes sense to use v16 instead of v18 because that's what this repo uses to test.